app/src/main/res/values/strings.xml

```diff
@@ -70,7 +70,7 @@
 
     <!-- TextsOnBlock -->
     <!-- Sequence -->
-    <string name="textOnBlocks.contentDefaultSec">0</string>
+    <string name="textOnBlocks.contentDefaultSec">0.0</string>
```

Comment:
well, this should be ``0`` because if the locale of the system is, for example ``France``, app will crash 
since French people use a comma as a decimal point.
I apologise that I didn't comment that things.